### PR TITLE
Upgrade OCP levels for CI tests

### PR DESCRIPTION
**Pull request by josephca**
Upgrade the OCP levels from 4.13/4.14/4.15 to 4.14/4.15/4.16

### CODE REVIEW

**Comment by josephca** [comment]
@subhashkhileri can you provide the review? Thanks!

File: ci-operator/jobs/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main-periodics.yaml [hunk 1]
```diff
@@ -231,28 +231,15 @@ periodics:
   labels:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-ocp-v4-14
-  reporter_config:
```
Comment (subhashkhileri) [inline ci-operator/jobs/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main-periodics.yaml:235]: @josephca, for 4-15 you are missing the `reporter_config` for slack alerts.

Comment (josephca) [reply ci-operator/jobs/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main-periodics.yaml:235]: Thanks. Updated.

**Comment by subhashkhileri** [comment]
/pj-rehearse periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-ocp-v4-15

**Comment by subhashkhileri** [comment]
/pj-rehearse periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-ocp-v4-15

**Comment by josephca** [comment]
/pj-rehearse periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-ocp-v4-15

**Comment by josephca** [comment]
/pj-rehearse periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly

**Comment by josephca** [comment]
/pj-rehearse periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly

**Pull request merged**